### ISSUE TITLE
Left nav text mismatch of "WCAG TF" to the name of the TF in other places "WCAG 2.x Backlog TF"

### ISSUE THREAD

**Issue by maryjom** [w3c/wai-website#853]
### Resource URL

https://www.w3.org/WAI/about/groups/task-forces/wcag2x-backlog/

### Description

The page title of this page, short name for the TF, and references to the TF in other pages (AG WG Task Forces, the auto-generated TF page) all call this the "WCAG 2.x Backlog TF". However the left navigation link is called "WCAG Task Force". 

Additionally there are other mentions in the content of this page that calls this the "WCAG TF" which should be WCAG 2.x Backlog TF".

### Resource Shortname

wai-groups
